docs/docs-content/vm-management/create-manage-vm/advanced-topics/migrate-vm-kubevirt.md

```diff
@@ -0,0 +1,118 @@
+   | **Cluster**                                              | The vSphere compute cluster of the VM to migrate.                                                                                                                                                                         |                                              |
+   | **ESXi Hypervisor**                                      | The IP address of the node corresponding to the VM to migrate. If you have selected the `vCenter + ESXi host` migration source type, you will need to provide ESXi credentials.                                           |                                              |
+   | **VM**                                                   | The VM to migrate from the selected host.                                                                                                                                                                                 |                                              |
+   | **Add Another VM?**                                      | Indicate whether you want to select multiple VM from the vSphere environment.                                                                                                                                             | `Y` / `n`                                    |
+   | **Add Another Host?**                                    | Indicate whether you would like to perform two migrations in the same configuration.                                                                                                                                      |                                              |
+   | **Destination Network Type**                             | The network that the VMs will be mapped to in the VMO cluster.                                                                                                                                                            | `pod` / `multus`                             |
+   | **Destination StorageClass**                             | The storage class on the destination that will be used to create the VM volumes.                                                                                                                                          |                                              |
+   | **Destination StorageClass Access Mode**                 | The configured access mode on the cluster storage class.                                                                                                                                                                  | `ReadWriteOnce` / `ReadWriteMany`            |
+
+5. The migration begins as soon as you complete the configuration. Execute the following command to watch the migration
+   status.
+
+   ```shell
+   watch --interval 2 'kubectl --namespace konveyor-forklift get migrations.forklift.konveyor.io <migration-name>-migration'
```

Comment:
The `<migration-name>` is that something the user needs to replace?

Reply:
Called out that the user needs to replace this one.